playbooks/tasks/portal-configs-load.yml

```diff
@@ -43,6 +43,10 @@
   loop: "{{ webportal_common_config_defaults | dict2items }}"
   no_log: True
 
+- name: Merge missing values to the common/cluster config
```

Comment:
Follow up: Any time we move the order of things for a critical fix we should  definitely add a comment as to why the ordering is critical.